src/npc.cpp

```diff
@@ -1635,7 +1635,7 @@ bool npc::wants_to_sell( const item &it, int at_price, int market_price ) const
     }
 
     // TODO: Base on inventory
-    return at_price - market_price <= 50;
+    return at_price > 0;
```

Comment:
Why not `return at_price > market_price;`?

Reply:
I think this is always `true` in the current implementation (except `active explosives or spilled liquids`, it has magic `-1000`). However, logically (for the future), then the possibility of selling for less than the base price cannot be ruled out. For example, if you have a very very good relationship, or if the merchant has a lot of goods, or you use some discount (kind of a shotgun in the face -_^)).

Reply:
By the way, about spilled liquids and magic numbers. I have seen an exploit where the liquid is displayed for sale separately from the container (apparently this is due to the pocket system). And this liquid can be bought for a negative price. That is, we buy `AR-15` and cough syrup without a bottle and the merchant will pay us more to get rid of this terrible syrup :) I'm not sure, maybe it is already fixed.